content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "Monitoring as Code with Sensu"
+linkTitle: "Monitoring as Code"
+description: "Read this page to learn about following a monitoring-as-code approach with Sensu."
+product: "Sensu Go"
+version: "6.2"
+weight: 5
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.2:
+    parent: operations
+    identifier: monitoring-as-code
+---
+
+Sensu supports end-to-end monitoring as code so you can manage your observability and monitoring workflows the same way you build, test, and deploy your applications and infrastructure.
+Through the combination of composable building blocks and robust APIs, Sensu allows you to define your entire monitoring workflow as declarative YAML or JSON code in configuration files.
+Teams can share and remix monitoring workflows that include Sensu resources for collecting observability events and metrics, diagnosing issues, sending alerts, and automatically remediating problems.
+
+In the monitoring as code approach, when a new endpoint starts up, like a cloud compute instance or Kubernetes Pod, Sensu automatically registers itself with the platform and starts collecting monitoring and observability data according to the code in your configuration files.
+Manage your monitoring and observability workflow as code, the same way you manage the systems you're monitoring, to improve visibility, reliability, portability, and repeatability.
+
+- Share, edit, review, and version your Sensu workflow configuration files just like you would with other "as code" solutions, within one team or among teams across your organization.
+- Maintain revision control and change history for your monitoring workflows.
+- Export your Sensu configuration from one environment and initialize another environment with the same configuration.
+- Remove, restore, back up, and recover Sensu instances based on Sensu workflow configuration files.
+- Use `sensuctl prune` to consistently apply changes to your monitoring workflows with a single command.
+- Include your monitoring workflow in your centralized continuous integration/continuous delivery (CI/CD) pipeline to keep your monitoring aligned with your product and services.
+
+## Create a monitoring as code repository
+
+A monitoring as code repository will include configuration files that contain the resource definitions you use in your monitoring workflow.
+You can use any source control repository for maintaining your monitoring workflow configuration files.
+
```

Comment:
Maybe this should say something like "To get started with monitoring as code, you'll need configuration files that contain your resource definitions, a repository where you can store and maintain your configuration files, and a file structure and naming convention."